src/vs/workbench/contrib/extensions/browser/media/extension.css

```diff
@@ -181,9 +186,14 @@
 
 .extension-list-item > .details > .footer > .author > .publisher-name {
 	font-size: 90%;
+	color: var(--vscode-descriptionForeground);
```

Comment:
All looks good except that this might be causing this situation for disabled extensions. Removing the added rule fixes the issue for disabled items but obviously reverts your intended color for the active ones.

<img width="334" alt="CleanShot 2022-12-16 at 11 33 14@2x" src="https://user-images.githubusercontent.com/25163139/208175155-70d56617-2c19-4e7c-9015-10d83759fab1.png">
<img width="332" alt="CleanShot 2022-12-16 at 11 33 21@2x" src="https://user-images.githubusercontent.com/25163139/208175158-4d07dafb-f230-4162-8c07-b323ad8ed286.png">